mpitests/test_adjoint_jacobian.py

```diff
@@ -26,20 +26,19 @@
 from pennylane import QNode
 from pennylane import numpy as np
 from pennylane import qnode
+from pennylane.devices import DefaultExecutionConfig, DefaultQubit, ExecutionConfig, MCMConfig
+from pennylane.tape import QuantumScript
 from scipy.stats import unitary_group
 
+import pennylane_lightning
```

Comment:
Do we need all modules in `pennylane_lightning`?

Reply:
No, I will remove it :+1: